### ISSUE TITLE
Purple tag requires more contrast accessibility

### ISSUE THREAD

**Issue by angella-qian** [carbon-design-system/carbon#5200]
## Environment

> Operating system

MacOS

> Browser

Chrome

> Automated testing tool and ruleset

> Assistive technology used to verify

- [WebAIM](https://webaim.org/resources/contrastchecker/?fcolor=6929C4&bcolor=E8DAFF)
- [Contrast Checker](https://contrastchecker.com/?c=6929c4&b=e8daff)

## Detailed description

> What version of the Carbon Design System are you using?

"carbon-components": "10.7.4",
"carbon-components-react": "version": "7.7.4"

> What did you expect to happen?

Expect more contrast between text color and tag background

> What happened instead?

Not enough contrast

> What WCAG 2.1 checkpoint does the issue violate?

Success Criterion 1.4.6 Contrast (Enhanced) Level AAA for fonts below 18 pt

> Please create a reduced test case in CodeSandbox

https://codesandbox.io/s/nice-sound-cyevx

> Steps to reproduce:
1. Open reduced test case
2. Right click on purple tag > Inspect > Computed
2. See font-size, background-color, color

## Additional information
According to these websites, the purple tag fails WCAG `AAA` because font size is 12px aka 9pt (<18pt) and it also fails `363` for people with color blindness.
[WebAIM](https://webaim.org/resources/contrastchecker/?fcolor=6929C4&bcolor=E8DAFF):
><img width="1001" alt="Screen Shot 2020-01-24 at 12 24 48 PM" src="https://media.github.ibm.com/user/229682/files/c393f100-3ea5-11ea-9d7e-9d2c3ba929e5">

[Contrast Checker:](https://contrastchecker.com/?c=6929c4&b=e8daff)
><img width="999" alt="Screen Shot 2020-01-24 at 12 29 01 PM" src="https://media.github.ibm.com/user/229682/files/ae1ec700-3ea5-11ea-8dec-28a663036f3b">

[Web Semantics – Font Size Conversion](https://websemantics.uk/tools/font-size-conversion-pixel-point-em-rem-percent/):
><img width="571" alt="Screen Shot 2020-01-24 at 12 32 24 PM" src="https://media.github.ibm.com/user/229682/files/afe88a80-3ea5-11ea-9e0f-bb6d2034fcbd">

Related issue:
https://github.ibm.com/watson-engagement-advisor/wea-backlog/issues/33973

**Comment by joshblack** [comment]
Hi @angella-qian! 👋 It seems from your issue that this is a AAA requirement, is this correct? If so, unfortunately, we're going for WCAG 2.1 AA so we might not be able to accommodate AAA requirements.

**Comment by joshblack** [comment]
cc @aagonzales are we doing AAA contrast requirements or AA?

**Comment by aagonzales** [comment]
AA are the requirements we need to meet. 4.5:1 for text contrast and 3:1 for object/non-text contrast.

Closing because purple tags mets WCAG AA color contrast. Thanks for investigating!
